From: Michael Biebl <biebl@debian.org>
Date: Mon, 17 Mar 2014 04:03:26 +0100
Subject: Order remote-fs.target after local-fs.target

This change was part of the old debianisation branch created by Tollef
and reflects the fact that on Debian the $remote_fs system facility
depends on $local_fs.
---
 units/remote-fs.target | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/units/remote-fs.target b/units/remote-fs.target
index 43ffa5c..655621f 100644
--- a/units/remote-fs.target
+++ b/units/remote-fs.target
@@ -8,7 +8,7 @@
 [Unit]
 Description=Remote File Systems
 Documentation=man:systemd.special(7)
-After=remote-fs-pre.target
+After=remote-fs-pre.target local-fs.target
 DefaultDependencies=no
 Conflicts=shutdown.target
 
